### PR TITLE
hestia: Ability to specify SMTP account for server communication

### PR DESCRIPTION
**Pull request by Myself5**
(new PR to be in line with branch naming for contribution guidelines)

### CODE REVIEW

**Comment by Myself5** [comment]
6ef4233 fixes passing undefined variables to v-change-sys-config-value to prevent `Usage: v-change-sys-config-value KEY VALUE` errors in `v-setup-server-smtp-account`. Should be ready to go now.

**Comment by Lupul** [comment]
https://getcomposer.org/doc/faqs/should-i-commit-the-dependencies-in-my-vendor-directory.md

**Comment by jaapmarcus** [comment]
We should probally download them on install / update like we do as on Filegator.

**Comment by Myself5** [comment]
> [getcomposer.org/doc/faqs/should-i-commit-the-dependencies-in-my-vendor-directory.md](https://getcomposer.org/doc/faqs/should-i-commit-the-dependencies-in-my-vendor-directory.md)

almost saw that one coming, which is why it's a seperate commit :P
How would I pull the dependency afterwards though?

File: bin/v-setup-server-smtp-account [hunk 1]
```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```
Comment (jaapmarcus) [inline bin/v-setup-server-smtp-account]: We should move it to /install/update/manual/  and replace it this script so we can automate it later on via CLI / php

**Comment by Myself5** [comment]
As discussed, the API/CLI compliant scripts have been introduced and the User Interactice script was moved to /install/update/manual. Should be good to go now

**Comment by jaapmarcus** [comment]
Also the markup is lost:

Hello, Jaap,To reset your Hestia Control Panel password, please follow this link: https://xxxx:8083/reset/?action=confirm&user=jaap&code=$xxx Alternatively, you may go to https://xxxx:8083/reset/?action=code&user=jaap and enter the following reset code: $xxxx If you did not request password reset, please ignore this message and accept our apologies. -- Hestia Control Panel

**Review by jaapmarcus — APPROVED** [review]

**Pull request merged**